src/pages/Upload/UploadPage.jsx

```diff
@@ -58,29 +58,31 @@ export default function UploadPage() {
   };
   return (
     <Stack direction="column" spacing={3} className={styles.title}>
-      <Button variant="contained" onClick={handleClickOpen}>
-        Upload Timetable
+      <Button variant="contained" onClick={handleClickOpen} className={styles.uploadButton}>
+        <b className={styles.textColour}>Upload Timetable ICS File</b>
       </Button>
 
       <UploadPopUp open={open} close={handleClose} />
-      <Typography>OR</Typography>
-      <TextField
-        id="outlined-input"
-        aria-label="Enter URL"
-        label="Enter your calendar URL"
-        disabled={disable}
-        value={calendarURL}
-        onChange={handleChange}
-        InputProps={{
-          endAdornment: (
+      <Typography><b>OR</b></Typography>
+      <FormControl>
+        <InputLabel><b className={styles.textColour}>Input your timetable URL...</b></InputLabel>
+        <OutlinedInput
+          className={styles.textBox}
+          id="outlined-input"
+          aria-label="Enter URL"
+          disabled={disable}
+          value={calendarURL}
+          onChange={handleChange}
+          endAdornment={
             <InputAdornment position="end">
-              <IconButton edge="end" color="primary" aria-label="submit" onClick={handleSubmit}>
-                <SendIcon />
-              </IconButton>
+              <button type="button" variant="contained" aria-label="submit" onClick={handleSubmit} className={styles.button}>
+                <b className={styles.textColour}>Submit</b>
+              </button>
```

Comment:
A suggestion here, perhaps it's better if MUI's Button component was used to match the overall style of the component. The code changes in this file would be as follows:

\*Here, because the style is inline following MUI's styling guide, its className prop has become redundant and was thus removed. 

```suggestion
              <Button
                variant="contained"
                aria-label="submit"
                onClick={handleSubmit}
                sx={{
                  borderRadius: '30px',
                  padding: '10px',
                  paddingLeft: '20px',
                  paddingRight: '20px',
                  borderWidth: '1px',
                }}> 
                  <b className={styles.textColour}>Submit</b>
                </Button>
```

Following change would make it look like this:

<img width="592" alt="Screenshot 2022-04-02 오후 9 34 52" src="https://user-images.githubusercontent.com/68981068/161374909-396a36f6-ccac-4a78-bb6b-dd61e2c9dcdb.png">

Reply:
Yes, looks much better!

Reply:
Yep just did it. Have a look and let me know if there are any other changes that should be made.